pinot-common/src/test/java/org/apache/pinot/common/metrics/PinotJMXToPromMetricsTest.java

```diff
@@ -0,0 +1,379 @@
+  protected static final String LABEL_KEY_STATUS = "status";
+  protected static final String LABEL_VAL_STATUS_IN_PROGRESS = "IN_PROGRESS";
+
+  protected static final List<String> METER_TYPES =
+      List.of("Count", "FiveMinuteRate", "MeanRate", "OneMinuteRate", "FifteenMinuteRate");
+
+  protected static final List<String> TIMER_TYPES =
+      List.of("Count", "FiveMinuteRate", "Max", "999thPercentile", "95thPercentile", "75thPercentile", "98thPercentile",
+          "OneMinuteRate", "50thPercentile", "99thPercentile", "FifteenMinuteRate", "Mean", "StdDev", "MeanRate",
+          "Min");
+
+  protected static final String TABLE_NAME_WITH_TYPE =
+      TableNameBuilder.forType(TableType.REALTIME).tableNameWithType(LABEL_VAL_RAW_TABLENAME);
+
+  protected static final String KAFKA_TOPIC = "myTopic";
+  protected static final String PARTITION_GROUP_ID = "partitionGroupId";
+  protected static final String CLIENT_ID =
+      String.format("%s-%s-%s", TABLE_NAME_WITH_TYPE, KAFKA_TOPIC, PARTITION_GROUP_ID);
+  protected static final String TABLE_STREAM_NAME = String.format("%s_%s", TABLE_NAME_WITH_TYPE, KAFKA_TOPIC);
+  protected static final List<String> EXPORTED_LABELS_FOR_TABLE_NAME_TABLE_TYPE =
+      List.of(LABEL_KEY_TABLE, LABEL_VAL_RAW_TABLENAME, LABEL_KEY_TABLETYPE, LABEL_VAL_TABLETYPE_REALTIME);
+
+  protected static final List<String> EXPORTED_LABELS_FOR_RAW_TABLE_NAME =
+      List.of(LABEL_KEY_TABLE, LABEL_VAL_RAW_TABLENAME);
+
+  protected static final List<String> EXPORTED_LABELS_FOR_CLIENT_ID =
+      List.of(LABEL_KEY_PARTITION, PARTITION_GROUP_ID, LABEL_KEY_TABLE, LABEL_VAL_RAW_TABLENAME, LABEL_KEY_TABLETYPE,
+          TableType.REALTIME.toString(), LABEL_KEY_TOPIC, KAFKA_TOPIC);
+
+  protected static final List<String> EXPORTED_LABELS_FOR_PARTITION_TABLE_NAME_AND_TYPE =
+      List.of(LABEL_KEY_PARTITION, "3", LABEL_KEY_TABLE, LABEL_VAL_RAW_TABLENAME, LABEL_KEY_TABLETYPE,
+          TableType.REALTIME.toString());
+
+  protected static final List<String> EXPORTED_LABELS_FOR_TABLE_TABLETYPE_TASKTYPE =
+      List.of(LABEL_KEY_TABLE, LABEL_VAL_RAW_TABLENAME, LABEL_KEY_TABLETYPE, LABEL_VAL_TABLETYPE_REALTIME,
+          LABEL_KEY_CONTROLLER_TASKTYPE, LABEL_VAL_CONTROLLER_TASKTYPE_CHC);
+
+  protected static final List<String> EXPORTED_LABELS_FOR_TABLENAMEANDTYPE_AND_TASKTYPE =
+      List.of(LABEL_KEY_TABLE, LABEL_VAL_TABLENAME_WITH_TYPE_REALTIME, LABEL_KEY_CONTROLLER_TASKTYPE,
+          LABEL_VAL_CONTROLLER_TASKTYPE_CHC);
+
+  protected static final List<String> EXPORTED_LABELS_FOR_TASK_TYPE_AND_STATUS =
+      List.of(LABEL_KEY_STATUS, LABEL_VAL_STATUS_IN_PROGRESS, LABEL_KEY_CONTROLLER_TASKTYPE,
+          LABEL_VAL_CONTROLLER_TASKTYPE_CHC);
+
+  protected static final List<String> EXPORTED_LABELS_PERIODIC_TASK_TABLE_TABLETYPE =
+      List.of(LABEL_KEY_CONTROLLER_PERIODIC_TASK, LABEL_VAL_CONTROLLER_TASKTYPE_CHC, LABEL_KEY_TABLE,
+          LABEL_VAL_RAW_TABLENAME, LABEL_KEY_TABLETYPE, LABEL_VAL_TABLETYPE_REALTIME);
```

Comment:
It looks to me that most of these attributes are only used in one of the tests. For example, EXPORTED_LABELS_FOR_TASK_TYPE_AND_STATUS is only used in controller tests. Shouldn't it be declared in that class instead of here? By declaring them here we make this class larger than needed and it is more difficult for a reader to understand the scope of this attribute